.circleci/config.yml

```diff
@@ -100,6 +101,10 @@ jobs:
             source venv/bin/activate
             python attack_range.py --mode terraform --action destroy
           when: on_fail
+      - slack/status:
+          webhook: '${SLACK_WEBHOOK}'
+          fail_only: true
+          mentions: jhernandez
```

Comment:
@d1vious : maybe we want to remove mentions so that you don't get bombarded?

Reply:
good point!